code/_onclick/click_override.dm

```diff
@@ -71,8 +71,10 @@
 		if(isliving(target_atom))
 			var/mob/living/L = target_atom
 			if(user.a_intent == INTENT_DISARM)
+				add_attack_logs(user, L, "shocked and weakened")
 				L.Weaken(3)
 			else
+				add_attack_logs(user, L, "electrocuted[P.unlimited_power ? " with unlimited power" : null]")
```

Comment:
not sure if it is work making an exception for unlimited power gloves considering they are giga admin fuckery. also, "electrocuted" isn't very descriptive. perhaps "electrocuted with [unlimited] power gloves"